resotocore/resotocore/web/certificate_handler.py

```diff
@@ -155,12 +167,13 @@ def lookup(config: CoreConfig, db: StandardDatabase) -> CertificateHandler:
             log.debug("Found existing certificate in data store.")
             key = load_key_from_bytes(maybe_ca["key"].encode("utf-8"))
             certificate = load_cert_from_bytes(maybe_ca["certificate"].encode("utf-8"))
-            return CertificateHandler(config, key, certificate)
+            log.info(f"Using CA certificate from database: {cert_fingerprint(certificate)}")
+            return CertificateHandler(config, key, certificate, temp_dir)
         else:
             wo = "with" if args.ca_cert_key_pass else "without"
-            log.info(f"No ca certificate found - create a new one {wo} passphrase.")
             key, certificate = bootstrap_ca()
+            log.info(f"No ca certificate found - create a new one {wo} passphrase: {cert_fingerprint(certificate)}")
```

Comment:
What's the purpose of this? Specifically the change where the certs fingerprint is printed as the passphrase?

Reply:
Just added logging of fingerprint. Added a dot to make it more clear.